include/bluetooth/audio/aics.h

```diff
@@ -0,0 +1,450 @@
+ *                     or ERRNO on negative value.
+ *                     For notifications, this will always be 0.
+ * @param active       Whether the instance is active or inactive.
+ */
+typedef void (*bt_aics_status_cb_t)(struct bt_conn *conn, struct bt_aics *inst, int err,
+				    bool active);
+
+/**
+ * @brief Callback function for the description.
+ *
+ * Called when the value is read, or if the value is changed by either the server or client.
+ *
+ * @param conn         Connection to peer device, or NULL if local server read.
+ * @param inst         The service pointer (as there may be multiple).
+ * @param err          Error value. 0 on success, GATT error on positive value
+ *                     or ERRNO on negative value.
+ *                     For notifications, this will always be 0.
+ * @param description  The description as a string (may have been clipped).
+ */
+typedef void (*bt_aics_description_cb_t)(struct bt_conn *conn, struct bt_aics *inst, int err,
+					 char *description);
+
+/**
+ * @brief Callback function for bt_aics_discover.
+ *
+ * This callback will usually be overwritten by the primary service that
+ * includes the Audio Input Control Service client.
+ *
+ * @param conn         Connection to peer device, or NULL if local server read.
+ * @param inst         The instance pointer.
+ * @param err          Error value. 0 on success, GATT error on positive value
+ *                     or ERRNO on negative value.
+ *                     For notifications, this will always be 0.
+ */
+typedef void (*bt_aics_discover_cb_t)(struct bt_conn *conn, struct bt_aics *inst, int err);
+
+struct bt_aics_cb {
+	bt_aics_state_cb_t                 state;
+	bt_aics_gain_setting_cb_t          gain_setting;
+	bt_aics_input_type_cb_t            type;
+	bt_aics_status_cb_t                status;
+	bt_aics_description_cb_t           description;
+
+#if defined(CONFIG_BT_AICS_CLIENT)
+	bt_aics_discover_cb_t              discover;
+	bt_aics_write_cb_t                 set_gain;
+	bt_aics_write_cb_t                 unmute;
+	bt_aics_write_cb_t                 mute;
+	bt_aics_write_cb_t                 set_manual_mode;
+	bt_aics_write_cb_t                 set_auto_mode;
+#endif /* CONFIG_BT_AICS_CLIENT */
+};
+
+
+/**
+ * @brief Discover a Audio Input Control Service.
+ *
+ * Attempts to discover a Audio Input Control Service on a server given the @p param.
+ *
+ * @param conn  Connection to the peer with the Audio Input Control Service.
+ * @param inst  Pointer to the Audio Input Control Service client instance.
+ * @param param Pointer to the parameters.
+ *
+ * @return 0 on success, ERRNO on fail.
+ */
+int bt_aics_discover(struct bt_conn *conn, struct bt_aics *inst,
+		     const struct bt_aics_discover_param *param);
+
+/**
+ * @brief Deactivates a Audio Input Control Service instance.
+ *
+ * Audio Input Control Services are activated by default, but this will allow
+ * the server deactivate a Audio Input Control Service.
```

Comment:
insert "to", replace "a" with "an" ("allow the server to deactivate an Audio [...]")